database/sqlite3.ksy

```diff
@@ -230,6 +230,7 @@ types:
         type: str
         size: serial_type.len_content
         encoding: UTF-8
+        if: serial_type.is_string
 #        if: _root.text_encoding == encodings::utf_8 and serial_type.is_string
```

Comment:
I wonder why the `if` was commented out 🤔 (do you see anything wrong with it?). I think it's more correct because you don't check which encoding the file uses and always parse the string as `UTF-8`, which seems to be incorrect.